Jenkinsfile

```diff
@@ -1,211 +1,16 @@
 @Library('socrata-pipeline-library')
 
-import com.socrata.ReleaseMetadataService
-def rmsSupportedEnvironment = com.socrata.ReleaseMetadataService.SupportedEnvironment
+Map pipelineParams = [
+  projectName: 'query-coordinator',
```

Comment:
I suggest sorting these keys to make finding keys easier.

Reply:
Yes.

Reply:
Done